### PR TITLE
Update IdP docs

### PR DESCRIPTION
**Pull request by jwitz**
Various fixes and clarifications to IdP related docs as requested by @aliotta

### CODE REVIEW

File: software/import-idp-groups.md [hunk 1]
```diff
@@ -69,4 +70,6 @@ By default, Astronomer assumes that the name of your group claim is `groups`. If
 
 ## Step 3: Add Teams to Workspaces and Deployments
 
-After configuring user groups on your IdP, Workspace Admins and Deployment Admins can configure those groups as Teams via the Astronomer UI. To learn more about adding and setting permissions for Teams via the Astronomer UI, read [User Permissions](workspace-permissions.md#via-teams).
+After configuring your IdP group, at least one user belonging to the group has to log in to Software before the group appears as a Team in the Software UI. To quickly import many groups at once, add a user to all groups and then log in to Software with the user.
```
Comment (kwcanuck) [inline software/import-idp-groups.md:73]:
```suggestion
After configuring your IdP group, at least one user belonging to the group has to log in to Software before the group appears as a Team in the Software UI. To quickly import many groups at once, add a user to all groups and then log in to Astronomer Software UI with the user's credentials.
```

Alternatively:

After configuring your IdP group, at least one user belonging to the group has to log in to Software before the group appears as a Team in the Software UI. To quickly import many groups at once, add a user to all groups and then log in to the Astronomer Software UI as the user.

File: software/import-idp-groups.md [hunk 2]
```diff
@@ -69,4 +70,6 @@ By default, Astronomer assumes that the name of your group claim is `groups`. If
...
+
+After configuring and importing user groups, Workspace Admins and Deployment Admins can configure those groups as Teams using the Astronomer UI. To learn more about adding and setting permissions for Teams via the Software UI, read [User Permissions](workspace-permissions.md#via-teams).
```
Comment (kwcanuck) [inline software/import-idp-groups.md:75]:
```suggestion
After configuring and importing user groups, Workspace Admins and Deployment Admins can configure those groups as Teams using the Astronomer Software UI. To learn more about adding and setting permissions for Teams in the Astronomer Software UI, see [User Permissions](workspace-permissions.md#via-teams).
```

File: software/import-idp-groups.md [hunk 3]
```diff
@@ -19,6 +19,7 @@ Before you implement Teams, consider the following:
 
 - By default, the first user to log in to your Astronomer platform is automatically granted `SYSTEM ADMIN` permissions. If you are configuring Teams for a new Astronomer installation, we recommend first logging in as the user who will be responsible for importing your IdP groups using Astronomer's default login flow.
 - Teams are based solely on the IdP group they were configured from, meaning that you cannot configure Team membership from Astronomer.
+- To remove a Team from your Software installation, you have to delete it from the Software UI or the Houston API. Deleting an IdP group doesn't automatically delete the associated Team.
```
Comment (kwcanuck) [inline software/import-idp-groups.md:22]:
```suggestion
- To remove a Team from your Software installation, you have to delete it from the Astronomer Software UI or the Houston API. Deleting an IdP group doesn't automatically delete the associated Team.
```

**Review by kwcanuck — APPROVED** [review]

File: software/import-idp-groups.md (hunk 3 above)
Comment (aliotta) [reply software/import-idp-groups.md:22]: Deleting an IdP group from the iDP UI doesn't automatically delete the associated Team from astronomer.

**Pull request merged**